studio/src/windowManagement/contentWindows/ContentWindowBuildView/EntryPointPopover.js

```diff
@@ -38,34 +39,37 @@ export async function getSelectedEntryPoint(projectSettingsManager, contentWindo
 	return entryPoints[0] || null;
 }
 
-export class EntryPointManager {
-	#projectSettings;
-	#assetManager;
-	#persistentData;
+export class EntryPointPopover extends Popover {
+	/** @type {import("../../../projectSelector/ProjectSettingsManager.js").ProjectSettingsManager?} */
+	#projectSettings = null;
+
+	/** @type {import("../../../assets/AssetManager.js").AssetManager?} */
+	#assetManager = null;
+
+	/** @type {import("../../ContentWindowPersistentData.js").ContentWindowPersistentData?} */
+	#persistentData = null;
 
 	/** @type {HTMLElement?} */
 	#currentSelectorEl = null;
+
 	#selectorContainer;
 	#droppableGui;
 
 	/**
-	 * @param {import("../../../ui/popoverMenus/Popover.js").Popover} popover
-	 * @param {import("../../../projectSelector/ProjectSettingsManager.js").ProjectSettingsManager} projectSettingsManager
-	 * @param {import("../../../assets/AssetManager.js").AssetManager} assetManager
-	 * @param {import("../../ContentWindowPersistentData.js").ContentWindowPersistentData} persistentData
+	 * @param {ConstructorParameters<typeof Popover>} args
 	 */
-	constructor(popover, projectSettingsManager, assetManager, persistentData) {
-		this.#projectSettings = projectSettingsManager;
-		this.#assetManager = assetManager;
-		this.#persistentData = persistentData;
+	constructor(...args) {
+		super(...args);
 
 		this.#selectorContainer = document.createElement("div");
-		popover.el.appendChild(this.#selectorContainer);
+		// TODO: double check "this" works
```

Comment:
I'm not sure what this `TODO` means, I tried it out on https://pr-502.renda.studio/ and it seems to be working.
Is there a circumstance that hasn't been tested yet?

Reply:
Whoops, that was just a mental note to double check the `Popover` class properties. Forgot to remove it!